code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -254,15 +254,14 @@
 	metabolization_rate = 2.5 * REAGENTS_METABOLISM
 	overdose_threshold = 100
 
-/datum/reagent/medicine/silver_sulfadiazine/reaction_mob(mob/living/M, method=TOUCH, reac_volume, show_message = 1, touch_protection, obj/item/bodypart/affecting)
+/datum/reagent/medicine/silver_sulfadiazine/reaction_mob(mob/living/M, method=TOUCH, reac_volume, show_message = 1) //NSV 13 - reverted to old application
```

Comment:
Please note that all of these meds you've changed here are actually topical applications IRL and really do only affect the area they are applied to.

Reply:
I am aware, thank you.